### ISSUE TITLE
Error: object ‘asRle’ is not exported by 'namespace:MsCoreUtils'

### ISSUE THREAD

**Issue by cpanse** [rformassspectrometry/Spectra#87]
using https://renkulab.io/projects/cp/hello-world

```
> BiocManager::install(c('rformassspectrometry/MsCoreUtils','rformassspectrometry/Spectra','cpanse/MsBackendRawFileReader'))
Bioconductor version 3.10 (BiocManager 1.30.10), R 3.6.1 (2019-07-05)
Installing github package(s) 'rformassspectrometry/MsCoreUtils',
  'rformassspectrometry/Spectra', 'cpanse/MsBackendRawFileReader'
Skipping install of 'MsCoreUtils' from a github remote, the SHA1 (f5a8d6e5) has not changed since last install.
  Use `force = TRUE` to force installation
Downloading GitHub repo rformassspectrometry/Spectra@master
✔  checking for file ‘/tmp/Rtmpl3hNPH/remotesa01b084b5a/rformassspectrometry-Spectra-53a46f6/DESCRIPTION’ ...
─  preparing ‘Spectra’:
✔  checking DESCRIPTION meta-information ...
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
─  building ‘Spectra_0.3.0.tar.gz’
   
* installing *source* package ‘Spectra’ ...
** using staged installation
** R
** byte-compile and prepare package for lazy loading
Error: object ‘asRle’ is not exported by 'namespace:MsCoreUtils'
Execution halted
ERROR: lazy loading failed for package ‘Spectra’
* removing ‘/usr/local/lib/R/site-library/Spectra’
Error: Failed to install 'Spectra' from GitHub:
  (converted from warning) installation of package ‘/tmp/Rtmpl3hNPH/filea017b17848/Spectra_0.3.0.tar.gz’ had non-zero exit status
> 
```

**Comment by cpanse** [comment]
```{r}
BiocManager::install(c('rformassspectrometry/MsCoreUtils','rformassspectrometry/Spectra'))
BiocManager::install('cpanse/MsBackendRawFileReader')
```
install works and demo runs. thanks;

![Screenshot 2019-12-13 at 11 21 15](https://user-images.githubusercontent.com/4901987/70793103-e73e2480-1d9a-11ea-8180-3fc0c4377ab0.png)

**Comment by sgibb** [comment]
Please install the latest `Spectra` version from github. It is solved by merging #85
